ReactAndroid/src/androidTest/js/ScrollViewTestModule.js

```diff
@@ -72,19 +72,23 @@ var onItemPress = function(itemNumber) {
   ScrollListener.onItemPress(itemNumber);
 };
 
-var ScrollViewTestApp = createReactClass({
-  displayName: 'ScrollViewTestApp',
-  getInitialState: getInitialState,
-  onScroll: onScroll,
-  onItemPress: onItemPress,
-  onScrollBeginDrag: onScrollBeginDrag,
-  onScrollEndDrag: onScrollEndDrag,
+class ScrollViewTestApp extends React.Component ({
+  constructor() {
+    super();
+    this.displayName = 'ScrollViewTestApp';
+    this.state = getInitialState;
+    this.onScroll = onScroll;
+    this.onItemPress = onItemPress;
+    this.onScrollBeginDrag = onScrollBeginDrag;
+    this.onScrollEndDrag = onScrollEndDrag;
+  }
+  
 
   scrollTo: function(destX, destY) {
```

Comment:
This is invalid syntax. Should fix it to something like this:

```suggestion
  scrollTo(destX, destY) {
```